audmath/core/api.py

```diff
@@ -86,6 +86,183 @@ def db(
     return x
 
 
+def duration_in_seconds(
+        time: typing.Union[float, int, str, np.timedelta64],
+        sampling_rate: int = None,
+) -> np.floating:
+    r"""Time in seconds.
+
+    Converts the given time value to seconds.
```

Comment:
```suggestion
    Converts the given duration to seconds.
```

Reply:
I changed all remaining time names in docstring to duration.